src/utils/hooks/useSubmitApplication.tsx

```diff
@@ -4,22 +4,13 @@ import {
   TemplateElementCategory,
   useUpdateApplicationMutation,
 } from '../generated/graphql'
-import { FullStructure, ResponseFull, UseGetApplicationProps } from '../types'
+import { FullStructure, UseGetApplicationProps } from '../types'
 
 const useSubmitApplication = ({ serialNumber }: UseGetApplicationProps) => {
-  const [submitted, setSubmitted] = useState(false)
-  const [processing, setProcessing] = useState(false)
   const [error, setError] = useState('')
 
   const [applicationSubmitMutation] = useUpdateApplicationMutation({
-    onCompleted: () => {
-      setProcessing(false)
-      setSubmitted(true)
-    },
-    onError: (submissionError) => {
-      setProcessing(false)
-      setError(submissionError.message)
-    },
+    onError: (submissionError) => setError(submissionError.message),
   })
 
   const submitFromStructure = async (structure: FullStructure) => {
```

Comment:
Yeah, this is the one -- should probably rename it to just `submit` as the only reason it had this name was so as not to clash with the old `submit` method. (Should have been called submitNEW really).

Reply:
Done: 6269273